vendor/ember-cli-qunit/test-loader.js

```diff
@@ -1,6 +1,20 @@
 /* globals jQuery, QUnit, require, requirejs */
 
 jQuery(document).ready(function() {
+  Ember.Test.QUnit2Adapter = Ember.Test.QUnitAdapter.extend({
```

Comment:
I'd like to move this into the `ember-qunit` suite, and not add to `Ember.Test`. Basically it would be:

```js
import { QUnitAdapter } from 'ember-qunit';
```